### PR TITLE
fixes media, reply, mentions and minor ui bugs

### PR DESCRIPTION
**Pull request by irffanasiff**

### CODE REVIEW

File: packages/chat-sdk/src/components/MediaContent.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const MediaContent = ({
   const [modalOpen, setModalOpen] = useState(false);
   const theme = useCustomTheme();
   const { isXs } = useBreakpoints();
-
+  console.log("background - ", theme.custom.colors.background);
```
Comment (hkirat) [inline packages/chat-sdk/src/components/MediaContent.tsx:19]: remove log

File: packages/chat-sdk/src/components/MediaContent.tsx [hunk 2]
```diff
@@ -95,11 +95,18 @@ export const MediaContent = ({
...
+                position: "relative",
+                maxHeight: !isXs ? 260 : 180,
+                width: 1,
+              }}
+            >
               <video
                 style={{
-                  height: !isXs ? 270 : 180,
-                  maxWidth: !isXs ? 375 : 250,
+                  maxHeight: !isXs ? 260 : 180,
```
Comment (hkirat) [inline packages/chat-sdk/src/components/MediaContent.tsx:107]: Could you explain this logic a little (why heights use px and width uses `vw` ?

File: packages/chat-sdk/src/components/messageInput/MessageInput.tsx [hunk 3]
```diff
@@ -59,50 +59,100 @@ export const CustomAutoComplete = () => {
...
+          borderRadius: "12px",
+          marginLeft: 10,
+          marginRight: 10,
+          marginBottom: 4,
+          padding: activeSearch && 12,
+        }}
+      >
+        {activeSearch !== "" && activeSearch && activeSearch.length !== 0 && (
+          <>
+            {offlineMembers
```
Comment (hkirat) [inline packages/chat-sdk/src/components/messageInput/MessageInput.tsx:77]: Is `offlineMembers` defined now?
Might be an issue with the merge conflict. Check the build for more details

**Review by hkirat — CHANGES_REQUESTED** [review]

File: packages/chat-sdk/src/components/MediaContent.tsx (hunk 2 above)
Comment (irffanasiff) [reply packages/chat-sdk/src/components/MediaContent.tsx:107]: the height is fixed for media but it can go to any width but not larger than screen-size

**Pull request closed without merging**